packages/payload/src/queues/config/types/index.ts

```diff
@@ -39,6 +40,13 @@ export type RunJobAccessArgs = {
 
 export type RunJobAccess = (args: RunJobAccessArgs) => boolean | Promise<boolean>
 
+/**
+ * Define the order of job execution for each queue using a Payload sort string.
+ *
+ * FIFO would equal `createdAt` and LIFO would equal `-createdAt`.
+ */
+export type QueueProcessingOrder = Sort
```

Comment:
Is this exported externally? I don't know why we need it since you have types on `processingOrder` and jsdoc there.

Reply:
It's not exported externally. I added it for 2 reasons:
- Type is more descriptive than just "Sort"
- JSDocs automatically get applied wherever this type is used

Do you want me to remove it? I don't care too much

Reply:
> Type is more descriptive than just "Sort"

If I'm looking at `JobsConfig['processingOrder']` type and see `QueueProcessingOrder`, I have to then look at it just to see it is a plain old `Sort` type that we have in a ton of other places.

I think we just remove it.

Reply:
Done in https://github.com/payloadcms/payload/pull/11897/commits/4f30704d6479a551cf54b348e3151a0d341f3d73